rudder-web/src/main/webapp/templates-hidden/common-layout.html

```diff
@@ -208,7 +208,7 @@ <h4 class="modal-title">Participate in usage survey?</h4>
     </div>
   <hr class="spacer" />
 <div id="footer">
-  <p>Rudder v2.3 - &copy; <a href="http://www.normation.com/">Normation</a></p>
+  <p>Rudder v3.0 - &copy; <a href="http://www.normation.com/">Normation</a></p>
```

Comment:
well, in fact that does not bring anything for now safe potential pb with integration. In all case, it should not be done siliently here (was a mistake: forgotten test).